app/assets/stylesheets/admin/views/_attachments.scss

```diff
@@ -0,0 +1,9 @@
+.js-title,
```

Comment:
So i think a few things are required to get this to work. I think we can just apply this to the .rtl class 

```
.right-to-left {
  input,
  textarea {
    direction: rtl;
  }
}

```

Reply:
Good point! Have updated this.